x-pack/plugins/security_solution/server/endpoint/routes/resolver/entity.ts

```diff
@@ -64,19 +59,6 @@ export function handleEntities(): RequestHandler<unknown, TypeOf<typeof validate
                     values: _id,
                   },
                 },
-                {
-                  exists: {
```

Comment:
By removing these checks, Resolver should work even if the new 7.9 index mappings aren't present. @jonathan-buttner Is that the right description? 

Also, how should we mark this PR in release notes? Enhancement? or bug fix?

Reply:
> By removing these checks, Resolver should work even if the new 7.9 index mappings aren't present. @jonathan-buttner Is that the right description?

That's correct.

> Also, how should we mark this PR in release notes? Enhancement? or bug fix?

@benskelker @spong This PR addresses the issue where resolver wouldn't render without the steps documented by Ben. Now resolver should render if the user upgrades their stack to 7.9.2 even if they don't perform the `.siem-signals` index mapping migration. Do we need to update release notes for this bug fix? If so how do I go about doing that?

Reply:
Also to answer the original question I think it's probably a bug fix.